doc/DeveloperGuide.md

```diff
@@ -88,6 +93,30 @@ Use case ends.
 > 3a1. AddressBook shows an error message <br>
   Use case resumes at step 2
 
++**MSS**
+
+1. User requests to rename tag
+2. AddressBook shows a list of persons with the tag
+3. AddressBook requests for confirmation
```

Comment:
Use cases should depict the back and forth interaction between the user and the system. For every step the user does, the system should do something in return (and vice-versa).